tests/unit/functions/test_functions_date_functions.py

```diff
@@ -1,16 +1,18 @@
 import data_factory_testing_framework.functions.functions_date_implementation as date_functions
 import pytest
 from _pytest.mark import param
-from freezegun import freeze_time
+from data_factory_testing_framework.pythonnet.csharp_datetime import CSharpDateTime
 
 
-@freeze_time("2023-11-24T12:11:54.753132")
 def test_utcnow() -> None:
+    # Arrange
+    CSharpDateTime.utcnow = lambda: CSharpDateTime.parse("2023-11-24T12:11:54.7531321Z")
```

Comment:
While this manual monkeypatching works, it might be good using the `Monkeypatch` capabilities of pytest [here](https://docs.pytest.org/en/6.2.x/monkeypatch.html). It provides some utilities to safely patch methods.

Reply:
Good to know that this was actually unsafe, fixed!